.circleci/config.yml

```diff
@@ -164,9 +169,17 @@ jobs:
           command: |
             export IMAGE_TAG=$([[ "$CIRCLE_BRANCH" == "staging" ]] && echo "staging-candidate" || echo "discardable")
             export SNYK_MONITOR_IMAGE_TAG="${IMAGE_TAG}-${CIRCLE_SHA1}"
-            export SNYK_OPERATOR_VERSION="0.0.1"
+            export SNYK_OPERATOR_VERSION="0.0.1-${CIRCLE_SHA1}"
             export SNYK_OPERATOR_IMAGE_TAG="${SNYK_MONITOR_IMAGE_TAG}"
             ./scripts/package-operator.sh "${SNYK_OPERATOR_VERSION}" "${SNYK_OPERATOR_IMAGE_TAG}" "${SNYK_MONITOR_IMAGE_TAG}"
+      - run:
+          name: REMOVE TEMPLATED OPERATOR BEFORE PERSISTING TO WORKSPACE
+          command: |
+            rm -r snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0
```

Comment:
I usually prefer ```-f``` here just to avoid flakiness or errors. would that make sense here?

Reply:
🤷‍♂ So far no issues without `-f` but I'm happy to add it anyway!